src/applications/claims-status/reducers/claim-detail.js

```diff
@@ -24,6 +25,9 @@ export default function claimDetailReducer(state = initialState, action) {
     case GET_CLAIM_DETAIL: {
       return set('loading', true, state);
     }
+    case CLEAR_CLAIM_DETAIL: {
+      return initialState;
+    }
```

Comment:
Should this return `{...state, ...initialState}` instead? Is there other data in state besides `detail` and `loading` that's possibly being cleared out here? If you couldn't tell, my Redux is quite rusty 😅

Reply:
No I think it's fine to just set it to initialState